src/mpid/ch4/include/mpidpre.h

```diff
@@ -450,6 +450,8 @@ typedef enum {
     MPIDI_WINATTR_NM_REACHABLE = 16,    /* whether a netmod may reach the window. Set by netmod at win init.
                                          * It only indicates whether the win type is supported. Per-target check
                                          * may be required separately. */
+    MPIDI_WINATTR_NM_DYNAMIC_MEMS = 32, /* whether the memory region is registered dynamically. Valid only for
+                                         * dynamic window. Set by netmod. */
```

Comment:
I initially thought MR is a bit network-specific term (e.g., valid for IB but may do not exist for another networks). But I think it is fine if we consider memory region (MR) is a general term rather than OFI MR or IB MR.